src/main/scala/xebia/cakemix/AskTimeoutProvider.scala

```diff
@@ -17,7 +17,9 @@ trait AskTimeoutProvider {
 }
 
 object AskTimeoutProvider {
-  def fromConfig(config: Config, key: String): Timeout = {
+  private val AskTimeoutConfigKey = "xebia.cakemix.ask-timeout"
```

Comment:
Well, he does see it when he adds/downloads the dependency.

I agree being more active in open source is great for many reasons (including but not limited to marketing and recruitment), we should definitely work on encouraging that and 'guidelines' (when written correctly) could help there.